cpp/src/slice2cs/CsMetadataValidator.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+#ifndef SLICE_CS_METADATA_VALIDATOR_H
+#define SLICE_CS_METADATA_VALIDATOR_H
+
+#include "../Slice/Parser.h"
+
+namespace Slice
+{
+
+    // Validates the C# metadata using "cs:" prefix in the given unit.
+    void validateCsMetadata(const UnitPtr& unit);
+
```

Comment:
I don't think we normally have this extra padding here.
Also, it should use triple-slash to be a doc-comment.
And I wouldn't mention C# and cs right next to each other. One is sufficient IMO.
```suggestion
    /// Validates any metadata with a "cs:" prefix in the given unit.
    void validateCsMetadata(const UnitPtr& unit);
```